.github/docs/CONTRIBUTING_NO_BACK_LINKS.md

```diff
@@ -1,229 +0,0 @@
-# Contributing to Scribe-Data
```

Comment:
I didn't check what the conflicts are exactly, but do make sure to take a look and resolve what the conflicts may be, so we are able to merge :grin: 

Looks like the conflict is on this file, but since the file is getting deleted anyway, my guess is that could just be there are some upstream changes that need to get pull down to the source branch

Reply:
@wkyoshida, phew, the conflict has been resolved now. :D